lib/api/auth.dart

```diff
@@ -1,10 +1,21 @@
 import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:firebase_auth/firebase_auth.dart';
 
+CollectionReference users = FirebaseFirestore.instance.collection("Users");
+bool loggedUserIsManager = false;
+
 Future<bool> signIn(String email, String password) async {
   try {
-    await FirebaseAuth.instance
+    UserCredential credential = await FirebaseAuth.instance
         .signInWithEmailAndPassword(email: email, password: password);
+
+    loggedUserIsManager = false;
+    final snapshot = await users.doc(credential.user?.uid).get();
+    if (snapshot.exists) {
+      Map data = snapshot.data() as Map;
+      loggedUserIsManager = data["manager"] ?? false;
```

Comment:
Todos os usuários devem ter o campo `manager`, que vai ter `true` ou `false`

Reply:
Entendi seu ponto. Pra ser sincera,  eu deixaria, just in case. Segurança a mais.